gui/src/main/java/org/verapdf/cli/commands/VeraCliArgParser.java

```diff
@@ -460,4 +487,52 @@ private static ProcessType typeFromArgs(VeraCliArgParser parser) {
 			retVal = ProcessType.addProcess(retVal, ProcessType.FIX);
 		return retVal;
 	}
+
+	public static List<String> getBaseVeraPDFParameters(VeraCliArgParser cliArgParser) {
```

Comment:
Not mad keen on this, might a wrapper class around `VeraCliArgParser` work better?